app/models/work.rb

```diff
@@ -88,6 +88,30 @@ class Work < ApplicationRecord
     end
   end
 
+  sig { params(first_interval: ActiveSupport::Duration, subsequent_interval: ActiveSupport::Duration).void }
+  def self.send_first_draft_reminders(first_interval: nil, subsequent_interval: nil)
```

Comment:
Can you move all of this to a service object (or background job)?  A models responsibility is about persistence, and this code is about sending mails.

Reply:
ah yeah, will do, had intended to do that and then forgot to refactor...